### PR TITLE
❄️ [amp-story-player] Fix flakiness in e2e test

### LINKED ISSUE #33482: ❄️ Flaky test: story player navigation
Failure:
```
  1) story player navigation
       chrome
          Standalone environment 
            
             first story should be playing video:

      AssertionError: expected true to deeply equal false
      + expected - actual

      -true
      +false
      
      at _callee2$ (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:115:45)
      at tryCatch (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:61:40)
      at GeneratorFunctionPrototype.invoke [as _invoke] (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:329:22)
      at GeneratorFunctionPrototype.prototype.<computed> [as next] (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:94:21)
      at asyncGeneratorStep (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:3:103)
      at _next (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:5:194)
```
also
```
  1) story player navigation
       chrome
          Standalone environment 
            
             navigating to next story pauses the previous one:

      AssertionError: expected false to deeply equal true
      + expected - actual

      -false
      +true
      
      at _callee3$ (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:175:45)
      at tryCatch (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:61:40)
      at GeneratorFunctionPrototype.invoke [as _invoke] (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:329:22)
      at GeneratorFunctionPrototype.prototype.<computed> [as next] (build-system/tasks/e2e/node_modules/babel-regenerator-runtime/runtime.js:94:21)
      at asyncGeneratorStep (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:3:103)
      at _next (extensions/amp-story-player/0.1/test/test-e2e/test-amp-story-player-navigation.js:5:194)
```

https://app.circleci.com/pipelines/github/ampproject/amphtml/5535/workflows/6d8d30c7-953d-40f9-8d61-d69d4357c7ee/jobs/75994
https://app.circleci.com/pipelines/github/ampproject/amphtml/5498/workflows/7135fc9b-b4bc-4c17-84ba-c48b6114e661/jobs/75341
https://app.circleci.com/pipelines/github/ampproject/amphtml/5498/workflows/7135fc9b-b4bc-4c17-84ba-c48b6114e661/jobs/75341

Skip PR incoming

@ampproject/wg-stories

### PR DESCRIPTION
**Pull request by Enriqe**
Closes  #33482

* Increases timeouts to decrease flakiness .
* Imports extension scripts manually in tests.

When running `amp --compiled` and navigating to the test page (e.g. `http://localhost:8000/test/fixtures/e2e/amp-story-player/navigation.amp.html`) the local server fails to fetch dynamically installed extensions (see error logs below).

e.g. in `amp-story.js` we install the `amp-viewer-integration-0.1.js` here when it's not found in the doc:
https://github.com/ampproject/amphtml/blob/6616e578b8c257d82ab91fd99aa720fcdc4022b1/extensions/amp-story/1.0/amp-story.js#L1363

We didn't have to manually add these extensions to the test files, I believe this is a regression in the local server @ampproject/wg-infra 

Error logs when running `amp --compiled` and navigating to the test page (e.g. `http://localhost:8000/test/fixtures/e2e/amp-story-player/navigation.amp.html`):
```
extensions-impl.js:586 GET https://cdn.ampproject.org/rtv/012103241413001/v0/amp-viewer-integration-0.1.js net::ERR_ABORTED 404
g.preloadExtension @ extensions-impl.js:586
g.installExtensionForDoc @ extensions-impl.js:233
push.f.g.buildCallback @ amp-story.js:1353
(anonymous) @ custom-element.js:533
Promise.then (async)
Si.b.buildInternal @ custom-element.js:532
g.build @ resource.js:335
bk @ resources-impl.js:456
ak @ resources-impl.js:391
g.upgraded @ resources-impl.js:508
Si.b.yd @ custom-element.js:396
Si.b.ig @ custom-element.js:1283
Si.b.bd @ custom-element.js:1207
Si.b.upgrade @ custom-element.js:367
Zi @ custom-element-registry.js:106
Xi @ custom-element-registry.js:49
sm @ extensions-impl.js:381
(anonymous) @ extensions-impl.js:363
lf @ style-installer.js:71
rm @ extensions-impl.js:359
(anonymous) @ extensions-impl.js:331
g.addDocFactory @ extensions-impl.js:436
g.addElement @ extensions-impl.js:330
(anonymous) @ amp-story.js:2956
f @ amp-story.js:2955
g.registerExtension @ extensions-impl.js:154
(anonymous) @ runtime.js:186
Promise.then (async)
amp-story @ runtime.js:182
Hi @ chunk.js:200
Di.sf @ chunk.js:419
(anonymous) @ chunk.js:474
Promise.then (async)
Ki @ chunk.js:473
Di.Aa @ chunk.js:492
Di.runForStartup @ chunk.js:378
Ei @ chunk.js:69
xm @ runtime.js:290
b @ runtime.js:197
c.AMP.push @ runtime.js:229
(anonymous) @ amp-story-1.0.js:1

```

```
extensions-impl.js:586 GET https://cdn.ampproject.org/rtv/012103241413001/v0/amp-social-share-0.1.js net::ERR_ABORTED 404
g.preloadExtension @ extensions-impl.js:586
g.installExtensionForDoc @ extensions-impl.js:233
push.f.Ah.loadRequiredExtensions @ amp-story-share.js:411
push.f.Ah.loadProviders @ amp-story-share.js:361
push.f.Ah.build @ amp-story-share.js:260
mutate @ amp-story-share-menu.js:197
jm @ vsync-impl.js:469
g.Eh @ vsync-impl.js:426
requestAnimationFrame (async)
hm @ vsync-impl.js:392
g.Aa @ vsync-impl.js:386
g.run @ vsync-impl.js:188
g.measure @ vsync-impl.js:251
mutate @ mutator-impl.js:241
jm @ vsync-impl.js:469
g.Eh @ vsync-impl.js:426
requestAnimationFrame (async)
hm @ vsync-impl.js:392
g.Aa @ vsync-impl.js:386
g.run @ vsync-impl.js:188
g.measure @ vsync-impl.js:251
mutate @ mutator-impl.js:241
jm @ vsync-impl.js:469
g.Eh @ vsync-impl.js:426
requestAnimationFrame (async)
hm @ vsync-impl.js:392
g.Aa @ vsync-impl.js:386
g.run @ vsync-impl.js:188
g.measure @ vsync-impl.js:251
mutate @ mutator-impl.js:241
jm @ vsync-impl.js:469
g.Eh @ vsync-impl.js:426
requestAnimationFrame (async)
hm @ vsync-impl.js:392
g.Aa @ vsync-impl.js:386
g.run @ vsync-impl.js:188
g.runPromise @ vsync-impl.js:203
uj @ mutator-impl.js:206
g.measureMutateElement @ mutator-impl.js:163
g.measureMutateElement @ base-element.js:1019
g.mutateElement @ base-element.js:999
ll @ amp-story.js:1790
push.f.g.onResize @ amp-story.js:1766
push.f.g.buildCallback @ amp-story.js:530
(anonymous) @ custom-element.js:533
Promise.then (async)
Si.b.buildInternal @ custom-element.js:532
g.build @ resource.js:335
bk @ resources-impl.js:456
ak @ resources-impl.js:391
g.upgraded @ resources-impl.js:508
Si.b.yd @ custom-element.js:396
Si.b.ig @ custom-element.js:1283
Si.b.bd @ custom-element.js:1207
Si.b.upgrade @ custom-element.js:367
Zi @ custom-element-registry.js:106
Xi @ custom-element-registry.js:49
sm @ extensions-impl.js:381
(anonymous) @ extensions-impl.js:363
lf @ style-installer.js:71
rm @ extensions-impl.js:359
(anonymous) @ extensions-impl.js:331
g.addDocFactory @ extensions-impl.js:436
g.addElement @ extensions-impl.js:330
(anonymous) @ amp-story.js:2956
f @ amp-story.js:2955
g.registerExtension @ extensions-impl.js:154
(anonymous) @ runtime.js:186
Promise.then (async)
amp-story @ runtime.js:182
Hi @ chunk.js:200
Di.sf @ chunk.js:419
(anonymous) @ chunk.js:474
Promise.then (async)
Ki @ chunk.js:473
Di.Aa @ chunk.js:492
Di.runForStartup @ chunk.js:378
Ei @ chunk.js:69
xm @ runtime.js:290
b @ runtime.js:197
c.AMP.push @ runtime.js:229
(anonymous) @ amp-story-1.0.js:1

```

### CODE REVIEW

**Review by gmajoulet — APPROVED** [review]

File: examples/amp-story/video-one-page.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
   <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
   <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0/amp-viewer-integration-0.1.js"></script>
+  <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script>
```
Comment (rsimha) [inline examples/amp-story/video-one-page.html:13]: > We didn't have to manually add these extensions to the test files, I believe this is a regression in the local server

> Error logs when running `amp --compiled` and navigating to the test page

@Enriqe the two things mentioned above are separate from each other.

- Running `amp --compiled` is a local-development-only scenario that lazy-builds extensions as they are fetched by test pages. If a test page fetches an extension or a binary, it will get lazily-built. Otherwise, it will not. (This fact has been true for a long time.)
- When unit / integration / e2e tests are run during CI, we pre-build the AMP runtime and all extensions. So it is guaranteed that binaries like `amp-story-1.0.js`, `amp-video-0.1.js`, `amp-viewer-integration-0.1.js`, and `amp-social-share-0.1.js` are all present in the `dist/` directory. The responsibility of actually fetching them lies with the test pages or the extensions themselves (in case one extension / JS binary auto-fetches another).

**TL;DR:** Lazy-building does not happen during CI, and is unlikely to have anything do to with flakiness. Curious: Were there other symptoms you noticed during CI testing that could point to what changed?
